frontend/src/components/footer.css

```diff
@@ -49,7 +49,7 @@ footer {
 
 .socials a {
     text-decoration: none;
```

Comment:
die müsste weg können, da du in der index.css gesagt hast, dass alle a elemente keine text-decoration haben?